### PR TITLE
Explain what ESC does

### LINKED ISSUE #82474: [rustdoc]: Explain what ESC does in the help box
I think we could include a text to explain what ESC does after #82234 merged.
image

This helps increasing popularity of the feature. Also, should we need to tag relnotes for the above PR ?

### PR DESCRIPTION
**Pull request by TaKO8Ki**
This pull request adds a text to explain what ESC does in doc.

closes https://github.com/rust-lang/rust/issues/82474

### CODE REVIEW

File: src/librustdoc/html/static/main.js [hunk 1]
```diff
@@ -2913,6 +2913,7 @@ function defocusSearchBar() {
             ["&#9166;", "Go to active search result"],
             ["+", "Expand all sections"],
             ["-", "Collapse all sections"],
+            ["esc", "Close the search view"],
```
Comment (GuillaumeGomez) [inline src/librustdoc/html/static/main.js:2916]: It does a few other things like:

 * Removing the query parameters from the URL (like `?whatever=12`)
 * Closing the help popup
 * Closing the theme list if open
 * Blurring the focus from the search input

**Comment by GuillaumeGomez** [comment]
I'm not a big fan of this listing, even more since I'm sure it might do more things in the future (and I very likely forgot some of them...). However I don't have an idea on how to improve it... Maybe some from @rust-lang/rustdoc has an idea?

**Comment by camelid** [comment]
I also don't feel this change makes sense. <kbd>esc</kbd> here doesn't do anything out of the ordinary; people expect <kbd>esc</kbd> to *escape* from a particular view. If we were using <kbd>esc</kbd> like Emacs, as a control key, then I think it should be documented, but none of these seem particularly helpful to document to me.

**Comment by TaKO8Ki** [comment]
@camelid @GuillaumeGomez @jyn514 
I think "↑" and "↓" don't do anything out of the ordinary, but they are documented. Considering that, I think esc should also be documented. Is it not necessary to explain what esc does?

**Comment by TaKO8Ki** [comment]
@GuillaumeGomez 
Hmm. I think different people have different thoughts and we should define what is ordinary thing in this case. Anyway, I accepted your opinion. I'm going to close https://github.com/rust-lang/rust/issues/82474 and this pull request.

**Pull request closed without merging**